addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1169,6 +1169,49 @@ QUnit.test('Post a message containing an email address followed by a mention on
     );
 });
 
+QUnit.test('Mention a partner with special character (e.g. apostrophe \')', async function (assert) {
```

Comment:
Since you still have changes to make in this commit, would also wrap string of test name with backquote `.
That way, don't need to escape '. It's nice to just copy-paste name of test and find it easily.

Reply:
Good point